### PR TITLE
Set available CPU_COUNT

### PR DESCRIPTION
**Pull request by jschueller**
xref: https://github.com/conda/conda-build/pull/1149

### CODE REVIEW

File: recipe/run_conda_forge_build_setup_win [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+rem 2 cores available on Appveyor workers: https://www.appveyor.com/docs/build-configuration/
```
Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_win]: Could we please use `::`? Also let's add a line about when `CPU_COUNT` was added for pass through here too.

**Comment by jakirkham** [comment]
This needs a version bump. As this is a breaking change, we should go to `3.0.0`.

File: recipe/run_conda_forge_build_setup_win (hunk 1 above)
Comment (jschueller) [inline recipe/run_conda_forge_build_setup_win]: @jakirkham ok

**Comment by jschueller** [comment]
@jakirkham You sure this is breaking ? At worst the few recipes using CPU_COUNT will be slower. Maybe most do not.

**Comment by jakirkham** [comment]
Thanks @jschueller. LGTM.

Any other feedback?

**Comment by jakirkham** [comment]
> @jakirkham You sure this is breaking ? At worst the few recipes using CPU_COUNT will be slower. Maybe most do not.

I'm being very literal about it. We set `MAKEFLAGS` before and now we don't. Perhaps there could be argument that it isn't as bad of a break as one might think. Still it is a break.

**Comment by jschueller** [comment]
@jakirkham good

**Comment by jakirkham** [comment]
Thanks @jschueller.

**Comment by jakirkham** [comment]
Added an xref in the OP of the PR to the `conda-build` PR ( https://github.com/conda/conda-build/pull/1149 ) that made it possible to override `CPU_COUNT` externally.

File: recipe/run_conda_forge_build_setup_win [hunk 2]
```diff
@@ -1,3 +1,8 @@
+
+:: 2 cores available on Appveyor workers: https://www.appveyor.com/docs/build-configuration/#build-environment
```
Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_win:2]: This link appears to be gone. Here's the new one.

ref: https://www.appveyor.com/docs/build-environment/#build-vm-configurations

Comment (jakirkham) [reply recipe/run_conda_forge_build_setup_win:2]: Fixing in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/73 ).

**Pull request merged**